src/createServer.js

```diff
@@ -1,11 +1,252 @@
 'use strict';
 
-// const express = require('express');
+const express = require('express');
 
 function createServer() {
   // Use express to create a server
   // Add a routes to the server
   // Return the server (express app)
+  const app = express();
+
+  let users = [];
+  // let users = [
+  //   {
+  //     id: 1,
+  //     name: 'Misha',
+  //   },
+  //   {
+  //     id: 2,
+  //     name: 'Dasha',
+  //   },
+  //   {
+  //     id: 3,
+  //     name: 'Lesya',
+  //   },
+  // ];
+
+  app.get('/users', express.json(), async (req, res) => {
+    res.send(users);
+  });
+
+  app.get('/users/:id', async (req, res) => {
+    const { id } = req.params;
+
+    const user = users.find((targetUser) => targetUser.id === +id);
+
+    if (!user) {
+      res.sendStatus(404);
+
+      return;
+    }
+
+    res.send(user);
+  });
+
+  app.post('/users', express.json(), async (req, res) => {
+    const { name } = req.body;
+    const newId = Math.max(...users.map((item) => item.id), 0) + 1;
+
+    if (!name) {
+      res.sendStatus(400);
+
+      return;
+    }
+
+    const user = {
+      name,
+      id: newId,
+    };
+
+    users.push(user);
+
+    res.statusCode = 201;
```

Comment:
Instead of setting `res.statusCode = 201;`, use `res.status(201);` to set the status code. This is the recommended way to set status codes in Express.